docs/diagrams/RateActivity.puml

```diff
@@ -0,0 +1,71 @@
+@startuml
+ !include style.puml
+
+ box Logic LOGIC_COLOR_T1
+ participant ":LogicManager" as LogicManager LOGIC_COLOR
+ participant ":HobbyListParser" as HobbyListParser LOGIC_COLOR
+ participant ":RateCommandParser" as RateCommandParser LOGIC_COLOR
+ participant ":r:RateCommand" as RateCommand LOGIC_COLOR
+ participant ":CommandResult" as CommandResult LOGIC_COLOR
```

Comment:
Would it be better to name this as result, since one of the return arrows later returns result?